hydra-cluster/config/protocol-parameters.json

```diff
@@ -379,5 +379,37 @@
     "stakePoolDeposit": 500000000,
     "stakePoolTargetNum": 500,
     "treasuryCut": 0.2,
-    "utxoCostPerByte": 4310
+    "utxoCostPerByte": 4310,
+    "poolVotingThresholds": {
+      "committeeNormal": 0.51,
+      "committeeNoConfidence": 0.51,
+      "hardForkInitiation": 0.51,
+      "motionNoConfidence": 0.51,
+      "ppSecurityGroup": 0.51
+    },
+    "dRepVotingThresholds": {
+      "motionNoConfidence": 0.51,
+      "committeeNormal": 0.51,
+      "committeeNoConfidence": 0.51,
+      "updateToConstitution": 0.51,
+      "hardForkInitiation": 0.51,
+      "ppNetworkGroup": 0.51,
+      "ppEconomicGroup": 0.51,
+      "ppTechnicalGroup": 0.51,
+      "ppGovGroup": 0.51,
+      "treasuryWithdrawal": 0.51
+    },
+    "committeeMinSize": 0,
+    "committeeMaxTermLength": 200,
+    "govActionLifetime": 10,
+    "govActionDeposit": 1000000000,
+    "dRepDeposit": 2000000,
+    "dRepActivity": 20,
+    "constitution": {
+      "anchor": {
+        "url": "",
+        "dataHash": "0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
TODO: Make sure these values come from the same place as the conway-genesis config yaml file above